app/src/main/java/io/reark/rxgithubapp/advanced/data/stores/NetworkRequestStatusStore.java

```diff
@@ -35,36 +35,38 @@
 
 import io.reark.reark.pojo.NetworkRequestStatus;
 import io.reark.reark.utils.Log;
-import io.reark.reark.utils.Preconditions;
-import io.reark.rxgithubapp.advanced.data.schematicProvider.GitHubProvider;
+import io.reark.rxgithubapp.advanced.data.schematicProvider.GitHubProvider.NetworkRequestStatuses;
 import io.reark.rxgithubapp.advanced.data.schematicProvider.JsonIdColumns;
 import io.reark.rxgithubapp.advanced.data.schematicProvider.UserSettingsColumns;
 
+import static io.reark.reark.utils.Preconditions.checkNotNull;
+
 public class NetworkRequestStatusStore extends GsonStoreBase<NetworkRequestStatus, Integer> {
     private static final String TAG = NetworkRequestStatusStore.class.getSimpleName();
 
-    public NetworkRequestStatusStore(@NonNull ContentResolver contentResolver, @NonNull Gson gson) {
+    public NetworkRequestStatusStore(@NonNull final ContentResolver contentResolver, @NonNull final Gson gson) {
         super(contentResolver, gson);
     }
 
     @NonNull
     @Override
-    protected Integer getIdFor(@NonNull NetworkRequestStatus item) {
-        Preconditions.checkNotNull(item, "Network Request Status cannot be null.");
+    protected Integer getIdFor(@NonNull final NetworkRequestStatus item) {
+        checkNotNull(item);
```

Comment:
Do you mean with Preconditions.get? This class already contains get(), so it would have to be a bit ugly version like

  return Preconditions.get(item).getUri().hashCode();